bookwyrm/templatetags/bookwyrm_tags.py

```diff
@@ -41,6 +41,17 @@ def get_book_description(book):
     return book.description or book.parent_work.description
 
 
+@register.filter(name="is_shelf_type")
+def is_shelf_type(current_shelf, shelf_type):
+    """is this shelf a readthrough shelf?"""
+    readthrough = current_shelf in ["to-read", "reading", "read"]
```

Comment:
A more future-proof way to check this would be to check the boolean field `shelf.editable`

Reply:
Ohh yeah that is a lot simpler lol.